### PR TITLE
Fix previous implementation of success/fail messages

### PR DESCRIPTION
**Pull request by kendase3**
Addresses OVER-9059 by splitting signal/slots into success/fail

### CODE REVIEW

**Comment by kendase3** [comment]
Hiyeran tested my previous implementation and it didn't display the expected string for when the user successfully signed into a google account but that account had no meepo account.

**Comment by kendase3** [comment]
I tested this version with that case and it worked on my machine.

**Comment by kendase3** [comment]
self-assigning for a second.  feel free to take a look but i'm looking over the new logic specifically to see if there are error flow regressions (the no-sites case and the meepo internal fail() calls that progress to error screen)

**Comment by kendase3** [comment]
(raises paw)

File: src/meepo.h [hunk 1]
```diff
@@ -72,6 +73,7 @@ class Meepo : public QObject {
   Sites sites_;
   QString error_;
   int sites_remaining_ = 0;
+  bool google_mode = false;
```
Comment (nicholasbishop) [inline src/meepo.h]: Should suffix with an underscore like the other fields

Comment (nicholasbishop) [reply src/meepo.h]: Also wouldn't hurt to add a comment here saying what google_mode is

**Pull request merged**